AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3210,6 +3210,7 @@ m.lavenir.net#$#main[role="main"] { transform: translateY(0px)!important; }
 !
 epl.delfi.ee#$?#.cookies-ribbon { remove: true; }
 xiacom.ru#?#body > div[id^="comp_"]:has(> div.meta-message > div.meta-message__text:contains(cookies))
+gymshark.com#?#.site-header > div.message-top-bar:has(> div.message-content:contains(cookies))
```

Comment:
Instead of double ext-CSS (contains) better use only one as ext-CSS:

.message-content > a[href^="/pages/cookie-policy"]